user/configure-device.rst

```diff
@@ -106,37 +72,46 @@ please add them):
         #option shared_secret ''
         #option consistent_key '1'
         #option mac_interface 'eth0'
+        #option management_interface 'tun0'
         #option merge_config '1'
         #option test_config '1'
         #option test_script '/usr/sbin/mytest'
-        #option uuid ''
-        #option key ''
-        list unmanaged 'system.@led'
-        list unmanaged 'network.loopback'
-        list unmanaged 'network.@switch'
-        list unmanaged 'network.@switch_vlan'
+        #option hardware_id_script '/usr/sbin/read_hw_id'
+        #option hardware_id_key '1'
+        option uuid ''
+        option key ''
         # curl options
         #option connect_timeout '15'
         #option max_time '30'
         #option capath '/etc/ssl/certs'
+        #option cacert '/etc/ssl/certs/ca-certificates.crt'
+        # hooks
+        #option pre_reload_hook '/usr/sbin/my_pre_reload_hook'
+        #option post_reload_hook '/usr/sbin/my_post_reload_hook'
 
 Uncomment and change the following fields:
 
 - ``url``: the hostname of your OpenWISP2 controller (for example, if
   you are hosting your OpenWISP server locally and you set the IP Address
-  to "192.168.56.2", the url would be ``https://192.168.56.2/``).
-- ``verify_ssl``: set to ``0`` if your controller's SSL certificate is
+  to "192.168.56.2", the url would be ``https://192.168.56.2``).
+- ``verify_ssl``: set to ``'0'`` if your controller's SSL certificate is
   self-signed; in production you will need a valid SSL certificate to
   keep your instance secure
 - ``shared_secret``: you can retrieve this from OpenWISP2 admin panel, in
   the Organization settings. The list of organizations is available at
   ``/admin/openwisp_users/organization/``.
 
+.. note::
+
+    For development or testing purposes you will usually use the local
```

Comment:
this applies only to developers who use the dev server, many users will use [ansible-openwisp2](https://github.com/openwisp/ansible-openwisp2) or [vagrant-openwisp](https://github.com/openwisp/vagrant-openwisp2), so we need to make this distinction clear or those who are not used to web / django development will get confused

Reply:
Once you get it working it's very straightforward however I added this part because I found confusing at the beginning which IP should I use, either the device IP or any other of the interfaces configured in the VM. I will reword this part though

Reply:
I think @nepython can confirm it ;)

Reply:
To restart `openwisp_config`, this also works,
```shell
service openwisp_config restart
```
It's easier to recall that's all :)